src/neofs_testlib/utils/wallet.py

```diff
@@ -1,8 +1,8 @@
 import json
 import logging
 
-from neo3.wallet import wallet as neo3_wallet
 from neo3.wallet import account as neo3_account
+from neo3.wallet import wallet as neo3_wallet
```

Comment:
Useless hunk, just don't change it please.

Reply:
this is from a black formatter, that configured like said in contribute.md. Remove it?

Reply:
Either drop, or make a separate formatting commit with no functional changes.

Reply:
Made a separate commit